### ISSUE TITLE
Batch Norm support

### ISSUE THREAD

**Issue by gits00** [iml130/nncg#5]
Hi, On using a batch normalization layer I get "Warning: BatchNormalization not implemented" but the [paper](https://arxiv.org/abs/2001.05572) mentions about using this layer in Table 3 wrt the testing done on Nao robot CNN. How can I add this layer in code?
Thanks.

**Comment by OliverUrbann** [comment]
This is correct. The paper is based on the [Code Generator](https://github.com/NaoDevils/CodeRelease/tree/master/Util/CNN/tfBallDetector) as released with the software for robot soccer by the team Nao Devils. NNCG is actually a reimplementation with a different software structure while generating the same code. However, the batch normalization in the Nao Devils repo is just merged with the convolution and thus must be the next layer of a convolution, see [here](https://github.com/NaoDevils/CodeRelease/blob/b47d2ce3474236325b9114913968b2eb39d0754e/Util/CNN/tfBallDetector/utility_functions/ocg.py#L398). This should also work here but we plan to implement a version that does not have this requirement.
